rosbag_filepath: "/home/victor/data/building_sim/terrain_coverage_2019-01-21-23-26-31.bag"
skip_first_n_sec: 50
map_evaluation:
  ground_truth_tsdf_layer_filepath: "/home/victor/data/voxgraph/ground_truth_maps/burning_building_rubble"